Lost Kingdoms/UniversalDIT_Flat_Pressed.json

```diff
@@ -0,0 +1,2136 @@
+      "`Axis 0+`": "../#DefaultDevices/DualShock4/Flat_Pressed/Left Stick.png",
+      "`Axis 1-`": "../#DefaultDevices/DualShock4/Flat_Pressed/Left Stick.png",
+      "`Axis 1+`": "../#DefaultDevices/DualShock4/Flat_Pressed/Left Stick.png",
+      "`Axis 2-`": "../#DefaultDevices/DualShock4/Flat_Pressed/Right Stick.png",
+      "`Axis 2+`": "../#DefaultDevices/DualShock4/Flat_Pressed/Right Stick.png",
+      "`Axis 3+`": "../#DefaultDevices/DualShock4/Flat_Pressed/Right Stick.png",
+      "`Axis 3-`": "../#DefaultDevices/DualShock4/Flat_Pressed/Right Stick.png",
+      "`Full Axis 4+`": "../#DefaultDevices/DualShock4/Flat_Pressed/L2.png",
+      "`Full Axis 5+`": "../#DefaultDevices/DualShock4/Flat_Pressed/R2.png"
+    },
+    "evdev/0/Sony Interactive Entertainment DualSense Wireless Controller": {
+      "evdev/0/Sony Interactive Entertainment DualSense Wireless Controller": "../#DefaultDevices/DualShock4/Device/Icon.png",
+      "SOUTH": "../#DefaultDevices/DualShock4/Flat_Pressed/Cross.png",
+      "EAST": "../#DefaultDevices/DualShock4/Flat_Pressed/Circle.png",
+      "WEST": "../#DefaultDevices/DualShock4/Flat_Pressed/Square.png",
+      "NORTH": "../#DefaultDevices/DualShock4/Flat_Pressed/Triangle.png",
+      "TL": "../#DefaultDevices/DualShock4/Flat_Pressed/L1.png",
+      "TR": "../#DefaultDevices/DualShock4/Flat_Pressed/R1.png",
+      "`TL2`": "../#DefaultDevices/DualShock4/Flat_Pressed/L2.png",
+      "`TR2`": "../#DefaultDevices/DualShock4/Flat_Pressed/R2.png",
+      "START": "../#DefaultDevices/DualShock4/Flat_Pressed/Options.png",
+      "SELECT": "../#DefaultDevices/DualShock4/Flat_Pressed/Share.png",
+      "MODE": "../#DefaultDevices/DualShock4/Flat_Pressed/PS.png",
+      "LEFT": "../#DefaultDevices/DualShock4/Flat_Pressed/Touchpad.png",
+      "`Axis 7-`": "../#DefaultDevices/DualShock4/Flat_Pressed/Pad.png",
+      "`Axis 7+`": "../#DefaultDevices/DualShock4/Flat_Pressed/Pad NS.png",
+      "`Axis 6+`": "../#DefaultDevices/DualShock4/Flat_Pressed/Pad EW.png",
+      "`Axis 6-`": "../#DefaultDevices/DualShock4/Flat_Pressed/Pad.png",
+      "`Axis 1+`": "../#DefaultDevices/DualShock4/Flat_Pressed/Left Stick.png",
+      "`Axis 1-`": "../#DefaultDevices/DualShock4/Flat_Pressed/Left Stick.png",
+      "`Axis 0+`": "../#DefaultDevices/DualShock4/Flat_Pressed/Left Stick.png",
+      "`Axis 0-`": "../#DefaultDevices/DualShock4/Flat_Pressed/Left Stick.png",
+      "THUMBL": "../#DefaultDevices/DualShock4/Flat_Pressed/L3.png",
+      "`Axis 4+`": "../#DefaultDevices/DualShock4/Flat_Pressed/Right Stick.png",
+      "`Axis 4-`": "../#DefaultDevices/DualShock4/Flat_Pressed/Right Stick.png",
+      "`Axis 3+`": "../#DefaultDevices/DualShock4/Flat_Pressed/Right Stick.png",
+      "`Axis 3-`": "../#DefaultDevices/DualShock4/Flat_Pressed/Right Stick.png",
+      "THUMBR": "../#DefaultDevices/DualShock4/Flat_Pressed/R3.png"
+    }
+  },
+  "generated_folder_name": "#Generated_",
```

Comment:
You must also set `generated_folder_name` for **UniversalDIT_Flat_Pressed** to `Generated_LostKingdoms`.